content/billing/managing-your-license-for-github-enterprise/downloading-your-license-for-github-enterprise.md

```diff
@@ -18,7 +18,13 @@ After you purchase or upgrade a license for {% data variables.product.prodname_e
 
 {% data reusables.enterprise-licensing.contact-sales-for-renewals-or-seats %}
 
-## Downloading your license from {% data variables.product.prodname_dotcom_the_website %}
+## Downloading your license if you don't have an enterprise account on {% data variables.product.prodname_dotcom_the_website %}
```

Comment:
We can trim this a bit, too ✂️

```suggestion
## Downloading your license without an enterprise account on {% data variables.product.prodname_dotcom_the_website %}
```

Reply:
I'd leave it as original. Otherwise it could be interpreted as a "license without enterprise account", which may create even more confusion.